diff --git a/node_modules/@amazon-devices/react-native-w3cmedia/dist/components/mediacontrols/FloatingButton.js b/node_modules/@amazon-devices/react-native-w3cmedia/dist/components/mediacontrols/FloatingButton.js
index 3aba614..da0fcc3 100644
--- a/node_modules/@amazon-devices/react-native-w3cmedia/dist/components/mediacontrols/FloatingButton.js
+++ b/node_modules/@amazon-devices/react-native-w3cmedia/dist/components/mediacontrols/FloatingButton.js
@@ -162,7 +162,7 @@ const styles = react_native_1.StyleSheet.create({
         fontSize: 50
     },
     pressedColor: {
-        color: 'rgba(43, 47, 53, 0.4)'
+        color: '#E50914'
     }
 });
 exports.default = FloatingButton;
diff --git a/node_modules/@amazon-devices/react-native-w3cmedia/dist/components/mediacontrols/FocusableButton.js b/node_modules/@amazon-devices/react-native-w3cmedia/dist/components/mediacontrols/FocusableButton.js
index 237565d..9df96e7 100644
--- a/node_modules/@amazon-devices/react-native-w3cmedia/dist/components/mediacontrols/FocusableButton.js
+++ b/node_modules/@amazon-devices/react-native-w3cmedia/dist/components/mediacontrols/FocusableButton.js
@@ -49,7 +49,7 @@ const DEFAULT_STYLE = react_native_1.StyleSheet.create({
         justifyContent: 'center'
     },
     BUTTON_FOCUSED: {
-        backgroundColor: '#2B2F35'
+        backgroundColor: '#E50914'
     },
     TITLE: {
         color: 'black',
